browser/src/NeovimInstance.ts

```diff
@@ -97,7 +97,7 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
 
         this._initPromise = startNeovim(this._pluginManager.getAllRuntimePaths(), filesToOpen)
             .then((nv) => {
-                console.log("NevoimInstance: Neovim started") // tslint:disable-line no-console
+                this.emit("logInfo", "NevoimInstance: Neovim started")
```

Comment:
Would you mind fixing my typo while you're here? 😄 I used `NevoimInstance` instead of `NeovimInstance`